.goreleaser.yaml

```diff
@@ -42,127 +49,59 @@ snapshot:
 
 changelog:
```

Comment:
Is this needed if we use `--changelog` with `make release-notes` output?

Reply:
Not needed, but I suppose it's better to have it for the sake of keeping `skip: false` in clear sight. Just added the skip to this section with the default comment so we know not to skip the changelog here either.